openfeature/openfeature.go

```diff
@@ -89,4 +89,5 @@ func RemoveHandler(eventType EventType, callback EventCallback) {
 // Shutdown active providers
 func Shutdown() {
 	api.Shutdown()
+	initSingleton()
```

Comment:
Please undo this change and any changes that do more than just remove `defer`. Other unrelated changes should be part of a separate PR. This change in particular clears the entire state of the API, including all registered providers, the global evaluation context, and all registered global hooks, which is definitely not the intention of `openfeature.Shutdown()`.

Reply:
I committed this mistakenly - this was some leftover code while I was debugging a separate issue. Thanks for catching this! Reverted.